tests/test_r2s.py

```diff
@@ -390,6 +392,17 @@ def test_tag_decay_time():
     m = tag_decay_time(m, decay_time)
     assert_array_equal(m.decay_time[m], decay_time)
 
+def test_photon_soruce_add_filetype():
```

Comment:
Still curious about this bit.

Reply:
This functions is not necessary now, and got removed in https://github.com/pyne/pyne/pull/1355/commits/e76741c7eae9ea492795461dd2ddc998bf75e19a.